Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.11.11-slim
+
+ENV PYTHONBUFFERED 1
+
+WORKDIR app/
```

Comment:
The `WORKDIR` command should not include a trailing slash. It should be `WORKDIR /app` instead of `WORKDIR app/` to ensure the correct directory is set as the working directory.